### ISSUE TITLE
RDS is in multi az and it is not stopping

### ISSUE THREAD

**Issue by lovepurohit** [aws-solutions/instance-scheduler-on-aws#136]
Hi,

I am trying to stop and start the RDS PostgreSQL server which is deployed in multi-az. When I check the logs, I could see that the instance have been fetched but it is not taking any action. 

Is there any way we could use the instance scheduler for the RDS which are in multi-az?

NOTE:
As there is an option to update the RDS to NON MULTI-AZ using CLI, Why can't we modify the RDS instance to multi-az set to no when stopping 
And when we start it, we can set the multi-az set to yes.

Thank you

**Comment by tapughose** [comment]
Hello @lovepurohit, thanks for reaching out. I was wondering if you have listed all the regions in `Region(s)` parameter while creating your stack.

**Comment by lovepurohit** [comment]
Hi @tapughose I have added all the regions but the issue in hand is, one of the RDS is in multi az and I have added Schedule tag on that instance but the instance scheduler is not performing on the resource.

The reason is multi az RDS instances cannot be stopped, but the instances can be modified to be set to non multi az when we want to stop the RDS.

So, I was wondering what is the use case of not modifying the RDS instances for non multi az?

Thank you
